templates/instagram.mustache

```diff
@@ -1,9 +1,9 @@
-<h3><a href="{{link}}" target="_blank">{{title}}</a></h3>
+<h3><a href="{{{link}}}" target="_blank">{{title}}</a></h3>
```

Comment:
`&` should be escaped as `&amp;` in HTML attributes

Reply:
You're right, got lost in all the escaping, unescaping, and sanitizing. The issue was double-escaping into `&amp;amp;`. Updated the PR and the description.